.github/workflows/rust.yml

```diff
@@ -11,13 +11,21 @@ env:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
     - uses: actions/checkout@v3
-    - name: Setup
-      run: sudo apt install llvm libclang-dev
+    - name: Setup Ubuntu
+      run: make setup-ubuntu
     - name: Run tests
-      run: cargo test --verbose
+      run: make ci
 
+  fuzz:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup fuzz
+      run: make -C fuzz setup-ubuntu
```

Comment:
Sure. CI responsiveness is something that needs care.
Let's correct that in the further work: https://github.com/Nitrokey/trussed-secrets-app/issues/85
It would be even better, if it would be simply packaged for Ubuntu on a PPA.
Resolving for now.

Reply:
Other option: the releases contain binaries https://github.com/rust-fuzz/cargo-fuzz/releases/tag/0.11.2

Reply:
Let's continue this discussion on a separate tickets, and push out this PR